### PR TITLE
Refactor to remove ct server related stuff

### PR DESCRIPTION
**Pull request by om-sharma**
We will not use ct xjson-server (https://github.com/openregister/mint/pull/64) which allows removal of ct related code. Removal of ct related code simplifies `writeEntriesInBatch` method of `DestinationDBUpdateDAO` class.

### CODE REVIEW

File: src/main/java/uk/gov/indexer/Configuration.java [hunk 1]
```diff
@@ -85,10 +85,6 @@ public String getProperty(String key) {
         return Optional.ofNullable(cloudSearchWatermarkEndPoints.get(register));
     }
 
-    public Optional<String> getCTServerEndpointForRegister(String register) {
-        return Optional.ofNullable(properties.getProperty(register + ".ctserver"));
-    }
-
     public Optional<String> elasticSerachEndPoint(String register) {
```
Comment (philandstuff) [inline src/main/java/uk/gov/indexer/Configuration.java]: just spotted a typo here `Serach`. unrelated to this PR but if you were to fix it I wouldn't mind :)

**Comment by philandstuff** [comment]
needs rebase

**Comment by om-sharma** [comment]
rebased and also corrected the spell mistake of a method name.

File: src/main/java/uk/gov/indexer/monitoring/CloudwatchRecordsProcessedUpdater.java [hunk 2]
```diff
@@ -21,7 +21,7 @@ public CloudwatchRecordsProcessedUpdater(String environment, String register) {
         acw.setEndpoint("https://monitoring.eu-west-1.amazonaws.com");
     }
 
-    public void update(long recordsProcessed) {
+    public void update(int recordsProcessed) {
```
Comment (philandstuff) [inline src/main/java/uk/gov/indexer/monitoring/CloudwatchRecordsProcessedUpdater.java:24]: why was this changed?

**Comment by philandstuff** [comment]
> +28 −768

:+1: :fire:

File: src/main/java/uk/gov/indexer/monitoring/CloudwatchRecordsProcessedUpdater.java (hunk 2 above)
Comment (om-sharma) [inline src/main/java/uk/gov/indexer/monitoring/CloudwatchRecordsProcessedUpdater.java:24]: Caller of this function earlier had long value but after refactoring caller got int value.
